R/qc_missed_cleavages.R

```diff
@@ -0,0 +1,58 @@
+#' Check number of missed cleavages
+#'
+#' Calculates the percentage of missed cleavages for each sample (by count).
+#'
+#' @param data A dataframe containing at least sample names, peptide or precursor identifiers and missed cleavage counts for each peptide or precursor.
+#' @param sample The column in the data dataframe containing the sample name.
+#' @param grouping The column in the data dataframe containing either precursor or peptide identifiers.
+#' @param missed_cleavages The column in the data dataframe containing the counts of missed cleavages per peptide or precursor.
+#' @param intensity Optional column containing the corresponding intensity values to each peptide or precursor.
+#' @param plot A logical indicating whether the result should be plotted.
+#'
+#' @return A data frame that contains the calculated percentage made up by the sum of all peptides or precursors containing the corresponding amount of missed cleavages.
+#' @import dplyr
+#' @importFrom ggplot2
+#' @importFrom magrittr %>%
+#' @importFrom forcats fct_inorder
```

Comment:
Add @importFrom rlang .data
this is for using the .data argument to prevent some notes you get when you run the check.
```suggestion
#' @importFrom ggplot2
#' @importFrom magrittr %>%
#' @importFrom forcats fct_inorder
#' @importFrom rlang .data
```